parallel_hashmap/phmap.h

```diff
@@ -3247,7 +3251,7 @@ class parallel_hash_set
             set.lazy_emplace_at(offset, std::forward<F>(f));
             set.set_ctrl(offset, H2(hashval));
         }
-        return iterator_at(offset);
+        return iterator(&inner, &sets_[0] + num_tables, set.iterator_at(offset));
```

Comment:
Oh, sorry about breaking this.  What about this instead?

```suggestion
        return make_iterator(&inner, set.iterator_at(offset));
```